tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -853,6 +851,10 @@ def testMaxPoolingGradWithArgmax(self):
       self.assertAllClose(out,
                           [11.0, 12.0, 0.0, 13.0, 0.0, 14.0, 0.0, 0.0, 0.0])
 
+  def testMaxPoolingGradWithArgmax(self):
+    for use_gpu in [True, False]:
+      self._testMaxPoolingGradWithArgmax(use_gpu)
+
   def testMaxPoolingGradGradWithArgmax(self):
     # MaxPoolWithArgMax is implemented only on CUDA.
     if not test.is_gpu_available(cuda_only=True):
```

Comment:
You can just get rid of this check instead.

Reply:
Removed.